core/shared/src/main/scala/org/http4s/HttpDate.scala

```diff
@@ -42,8 +43,9 @@ class HttpDate private (val epochSecond: Long) extends Renderable with Ordered[H
   def compare(that: HttpDate): Int =
     this.epochSecond.compare(that.epochSecond)
 
-  def toInstant: Instant =
-    Instant.ofEpochSecond(epochSecond)
+  def toDuration: FiniteDuration = epochSecond.seconds
```

Comment:
New public API. Or maybe it's too weird? `FiniteDuration` of epoch time is probably not the ideal way to express a moment in time, but Cats Effect `Clock#realTime` has normalized it.